### ISSUE TITLE
remove scatterplot facet section

### ISSUE THREAD

**Issue by ErinBecker** [datacarpentry/openrefine-socialsci#4]
[Episode 5](http://www.datacarpentry.org/openrefine-socialsci/05-numbers/) has a section on scatterplot facets. When teaching the same material from the Ecology lesson, I've always skipped this section as it doesn't seem useful to the learners. Should it be removed from this lesson (especially given the addition of the regular expressions episode)?

**Comment by gtlaflair** [comment]
I agree that the scatterplot matrix created by openrefine is not that useful. Could the same concept be covered using `spm()` or `GGally::ggpairs()` in the r-socialsci lesson? Or would it be too much content?

**Comment by PeterSmyth12** [comment]
I seem to have removed the scatter plot, not sure why it let me.
